injectLatestProps.js

```diff
@@ -12,7 +12,9 @@ exec(
     console.log(`stderr: ${stderr}`);
   }
 );
-exec("cp -r ./react-native-elements/website/docs/props ./src/content/Props"),
+exec(
+  "cp -r ./react-native-elements/website/docs/main/props ./src/content/Props"
```

Comment:
```suggestion
  "cp -r ./react-native-elements/website/docs/props ./src/content/Props"
```

Reply:
@pranshuchittora this file has some substantial changes in #97 please look there